src/search.py

```diff
@@ -1,42 +1,61 @@
-import numpy as np
-from matplotlib.axes import Axes
-from statistics import mean
+from re import T
 from users import User
+from beam import Beam
 from typing import Union, Sequence
-from plotting import plot_beams
+from plotting import Cell
 
-def send_beam(start_angle, end_angle, users, cell_radius):
+def send_beam(beam: Beam, users, cell_radius):
     ack = 0
     for user in users:
-        if (user.theta >= start_angle) and (user.theta <= end_angle) and (user.radius <= cell_radius):
-            ack = ack + 1
+        if (user.theta >= beam.start_angle) and (user.theta <= beam.end_angle) and (user.radius <= cell_radius):
+            ack = 1
     return ack
```

Comment:
I think we want to get an ack for each user in the cell so `ack = ack + 1` might be better.

Reply:
I agree